### PR TITLE
Update OIM_URLs to drop useless params

### PR DESCRIPTION
**Pull request by matyasselmeci**
I simplified the queries to remove parameters that are ignored by the new webapp.

### CODE REVIEW

File: config/gracc-request.toml [hunk 1]
```diff
@@ -37,6 +37,6 @@ dest_field = 'OIM_Site'
 regex = true
 
 [OIM_URLs]
-voinfo = 'https://my.opensciencegrid.org/vosummary/xml?summary_attrs_showfield_of_science=on&all_vos=on&active=on&active_value=1&oasis_value=1&sort_key=name'
-oimtopology = 'https://my.opensciencegrid.org/rgsummary/xml?summary_attrs_showhierarchy=on&summary_attrs_showwlcg=on&summary_attrs_showservice=on&summary_attrs_showfqdn=on&summary_attrs_showvoownership=on&summary_attrs_showcontact=on&gip_status_attrs_showtestresults=on&downtime_attrs_showpast=&account_type=cumulative_hours&ce_account_type=gip_vo&se_account_type=vo_transfer_volume&bdiitree_type=total_jobs&bdii_object=service&bdii_server=is-osg&all_resources=on&facility_sel%5B%5D=10009&gridtype=on&gridtype_1=on&active=on&active_value=1&disable_value=1'
-projects = 'https://my.opensciencegrid.org/miscproject/xml?count_sg_1&count_active=on&count_enabled=on'
+voinfo = 'https://my.opensciencegrid.org/vosummary/xml?active=on&active_value=1&oasis_value=1'
```
Comment (matyasselmeci) [inline config/gracc-request.toml:40]: `oasis_value=1` does nothing if you don't also have `oasis=on`; do you want to change it?

File: config/gracc-request.toml [hunk 2]
```diff
@@ -37,6 +37,6 @@ dest_field = 'OIM_Site'
...
+oimtopology = 'https://my.opensciencegrid.org/rgsummary/xml?facility_sel%5B%5D=10009&gridtype=on&gridtype_1=on&active=on&active_value=1&disable_value=1'
```
Comment (matyasselmeci) [inline config/gracc-request.toml]: `disable_value=1` does nothing without `disable=on` (and `1` would make it show _only_ disabled resources); do you want to remove it or change it to `disable=on&disable_value=0`?

**Comment by matyasselmeci** [comment]
Note that the queries as written filter out resources that aren't _active_, as opposed to resources that are _disabled_. Is this what you want? I thought we've been telling people to set `active=false` instead of `disabled=true` because disabled == deleted as far as GRACC is concerned...

**Comment by matyasselmeci** [comment]
The CI tests seem to be unrelated to this change (I got the same failures when I reverted my commit). When I used curl, the results from the new queries were identical to the results from the old queries...

**Comment by djw8605** [comment]
I think the issue is that the "tested" resource, `AGLT2_SL6` has gone from active=True to active=False since the last test was run. (or the disable has been changed)

Could you modify the query such that it will get ALL resources, no matter active or disable state?  I believe that is what we want anyways.

**Comment by matyasselmeci** [comment]
The new query picks up AGLT2_SL6 but the test is still failing. Any ideas?

**Comment by djw8605** [comment]
I just committed a fix.  There are a few other references to URL's.  The tests use a special config in `tests` directory.  And there's a "default" url.

**Comment by djw8605** [comment]
Great, merging!

**Review by djw8605 — APPROVED** [review]
LGTM

**Pull request merged**